megamek/src/megamek/common/options/GameOptions.java

```diff
@@ -55,6 +55,7 @@ public GameOptions() {
     public synchronized void initialize() {
         IBasicOptionGroup base = addGroup("basic"); //$NON-NLS-1$
         addOption(base, OptionsConstants.BASE_FRIENDLY_FIRE, false); //$NON-NLS-1$
+        addOption(base, OptionsConstants.BASE_TURN_TIMER, 0); //$NON-NLS-1$
```

Comment:
I would prefer newer options go at the bottom, unless there are similar options for it to be grouped with. People get used to looking in a given option in a certain place.

Reply:
ok, will put new stuff to bottom in future, thanks for the advise.